composer.json

```diff
@@ -38,6 +38,7 @@
         "mockery/mockery": "^1.0",
         "orchestra/database": "3.5.*|3.6.*|3.7.*|3.8.x-dev",
         "orchestra/testbench": "3.5.*|3.6.*|3.7.*|3.8.*",
+        "phpstan/phpstan": "^0.11.5",
```

Comment:
In this case, should this required in travis file only?

Reply:
Done